neurom/features/morphology.py

```diff
@@ -55,11 +55,60 @@
 from neurom.core.types import NeuriteType
 from neurom.features import feature, NameSpace, neurite as nf
 from neurom.utils import str_to_plane
-
+from neurom.core.morphology import Neurite
+from collections import OrderedDict
```

Comment:
I think `dict`s are ordered by default fm 3.7 onward?

Reply:
Indeed. I thought it was an implementation detail but I checked there is an insertion order guarantee from 3.7